### DISCUSSION TITLE
Trigger error Playback Events by BT temperature

### DISCUSSION THREAD

**Discussion by Terracotta-6** [artisan-roaster-scope/artisan#1698] [General]
![b9c9a3ef2b0401be0dca8d4e2abe29a](https://github.com/user-attachments/assets/8011ae52-b91e-4994-8b04-1660f716be79)

**Comment by MAKOMO** [comment]
Unclear what the issue might be without that background profile and the settings that are used here. Please provide both. Thanks!

**Reply by Terracotta-6** [reply]
Could you please provide your email? Thank you.

**Reply by MAKOMO** [reply]
Please write to info at artisan.plus. Thanks!

**Comment by Terracotta-6** [comment]
An error occurred while saving the modified configuration
![aa9e95b17852d2b1d09be5f27ade692](https://github.com/user-attachments/assets/c042c3d1-9af3-4c2f-9f62-52e077bc9ac0)

**Reply by MAKOMO** [reply]
Could you please provide your artisan.log file to allow me to investigate this one? Hold the ALT button and click the plus icon. An email should open with that file attached. THANKS!

**Reply by Terracotta-6** [reply]
The reason is found. Even Types names cannot be empty when set.

**Reply by MAKOMO** [reply]
??

**Reply by Terracotta-6** [reply]
Did you get the mail?

**Reply by MAKOMO** [reply]
No, I did not get an email to info at artisan.plus yet.

**Reply by MAKOMO** [reply]
Please use the Artisan Contact Form. That way I should get your email address without you having to publish it here: https://artisan-roasterscope.blogspot.com/p/contact-me.html

**Reply by Terracotta-6** [reply]
The logic is perfectly sound!  Sorry again for wasting your precious time!
